src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -677,5 +677,23 @@ public void CheckTheCorrectRuntimeAssemblyIsUsedFromNuget ()
 				Assert.IsTrue (xab.LastBuildOutput.ContainsText (expected), $"Build should be using {expected}");
 			}
 		}
+
+		[Test]
+		public void MissingDexFile ()
+		{
+			//NOTE: The trailing / breaks <CompileToDalvik />
```

Comment:
Not important really, but should this be in the past tense like:

```
//NOTE: The trailing / broke <CompileToDalvik /> in d16-5
```

The dependence on `Trim ('\')` got into d16-5 sneakily via the relatively harmless-looking <https://github.com/xamarin/xamarin-android/commit/185c5297a296163eb9a64cc15488e8e829e26c89>.